cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -38,7 +38,7 @@ var addonsEnableCmd = &cobra.Command{
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on when it is applied.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on. Determines the order in which to apply the add-on.")
```

Comment:
@thatdocslady `Determines the order in which to apply the add-on.`  does not sound right. Also it makes it difficult to understand the actual meaning for sure.

Just to be clear (as @hferentschik mentioned) `priority` is something comes to play when we have enabled mutiple add-ons and we want the add-ons to apply in a specific order as there might be dependency issues.